src/library/cmd_reader.h

```diff
@@ -184,5 +163,160 @@ namespace xlang::cmd
         }
 
         std::map<std::string_view, std::vector<std::string>> m_options;
+
+		template<typename O, typename L>
+		void extract_option(std::string_view arg, O const& options, L& last)
```

Comment:
Consistency - use `const&` for `string_view` as that done elsewhere in this file.

Reply:
remove_prefix doesn't work with const& set. Should I change other methods to not be const& for consistency?